### PR TITLE
Adds support for meatchurch.com

### LINKED ISSUE #1884: I'd like to request support for https://www.meatchurch.com/
Sample recipe URLs:
https://www.meatchurch.com/blogs/recipes/baby-back-ribs

- Which version of recipe-scrapers are you using? Using Tandoor which is currently using 15.11.0
- How did you discover the package? Previous request to add site for support.
- What's your use case for recipe-scrapers? Tandoor

### PR DESCRIPTION
**Pull request by zdenek-stursa**
## Add scraper for meatchurch.com

Adds support for scraping recipes from meatchurch.com.

Closes #1884

### Site structure

Meat Church uses a Shopify-based blog layout with no JSON-LD Recipe schema — recipe content is embedded in plain HTML:

- Title: `<meta property="og:title">` or `<h1 class="section__title-text">`
- Ingredients: `<li>` items within `<ul>` elements inside `div.article__content`
- Instructions: `<p>` elements after the last `<ul>` in the article body
- Image: `<meta property="og:image:secure_url">`
- Site name / Author: hardcoded "Meat Church"

### Testing

Tested against the baby-back-ribs recipe from the issue.

All tests pass.

### Edge cases handled

- Ingredient lists include both required and optional ingredients (all `<ul>` items collected)
- Instruction paragraphs containing only images or whitespace are filtered out via `normalize_string`

### CODE REVIEW

File: recipe_scrapers/meatchurch.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            return ""
+
+        last_list = lists[-1]
+        steps = []
+        for sibling in last_list.next_siblings:
+            if sibling.name == "p":
+                text = normalize_string(sibling.get_text())
+                if text:
+                    steps.append(text)
+        return "\n".join(steps)
```
Comment (jknndy) [inline recipe_scrapers/meatchurch.py:48]:
```suggestion
        return "\n".join(steps)

    def total_time(self):
        raise FieldNotProvidedByWebsiteException(return_value=None)

    def yields(self):
        raise FieldNotProvidedByWebsiteException(return_value=None)

```

Because both `yields` and `total_time` are both [Mandatory Fields](https://docs.recipe-scrapers.com/contributing/in-depth-guide-scraper-functions/) so we should return an exception.

File: recipe_scrapers/meatchurch.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    def image(self):
+        og_image = self.soup.find("meta", {"property": "og:image:secure_url"})
+        if og_image:
+            return og_image.get("content")
+        og_image = self.soup.find("meta", {"property": "og:image"})
+        if og_image:
+            return og_image.get("content")
+        return None
+
```
Comment (jknndy) [inline recipe_scrapers/meatchurch.py]:
```suggestion
```

The default image gathering is returning the same url as this implementation is so we should favor the default.

File: recipe_scrapers/meatchurch.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        return "Meat Church"
+
+    def title(self):
+        og_title = self.soup.find("meta", {"property": "og:title"})
+        if og_title:
+            return normalize_string(og_title.get("content"))
+        heading = self.soup.find("h1", class_="section__title-text")
+        if heading:
+            return normalize_string(heading.get_text())
+        return None
```
Comment (jknndy) [inline recipe_scrapers/meatchurch.py]:
```suggestion
        return self.soup.find("meta", property="og:title")["content"]
```
I tested a few pages and the title always seems to be available at this location. Feel free to not make this change if you found an instance where this is not true.

File: tests/test_data/meatchurch.com/meatchurch.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "Return the ribs to the smoker and continue to cook until meat has started to pull back. You are looking for an internal temp of 203 - 205. This typically takes 90 mins or so depending on the size of the ribs. You can use Thermoworks Instant Read Thermometer to check the temp.",
+    "Remove the ribs from the foil and sauce or glaze them at this point.We use a mix of 3:1 mix of our favorite BBQ + Texas Pepper Jelly Rib Candy. TPJ has a lot of great flavor options (Apple n Brown Sugar or Peachy Peach). Feel free to make this recipe your own by choosing sauces you love.",
+    "It is only necessary to leave the ribs on the cooker for 10 minutes to set the sauce. This will keep the bbq sauce from running down your face. I love to sauce in the foil boat on the pit to keep the pit clean.",
+    "Remove the ribs from the smoker. Allow to sit for a few minutes. Then slice and enjoy!"
+  ],
```
Comment (jknndy) [inline tests/test_data/meatchurch.com/meatchurch.json:30]:
```suggestion
  ],
  "yields": null,
  "total_time": null,
```

**Review by jknndy — DISMISSED** [review]
Hi @zdenek-stursa , thanks for the contribution! I made a few comments for your review.
